pkg/operator/k8sutil/node.go

```diff
@@ -28,6 +29,11 @@ import (
 	helper "k8s.io/kubernetes/pkg/apis/core/v1/helper"
 )
 
+var validTopologyLabelKeys = []string{
+	"failure-domain.beta.kubernetes.io",
+	"failure-domain.kubernetes.io",
```

Comment:
`failure-domain.kubernetes.io` was introduced as a "well-known label" in 1.13. Not sure if it was ever officially promoted out of beta, though, since there's this KEP as well: https://github.com/kubernetes/enhancements/pull/839